### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2026' in position 11744: ordinal not in range(128)

### ISSUE THREAD

**Issue by nicloay** [grow/grow#210]
Hello. I'm using MacOSX with python 2.7.11
I'm truing to build project but getting this error

```
nicloay:paintcraft nicloay$ python --version
Python 2.7.11
nicloay:paintcraft nicloay$ grow build paintcraft.in/
[16:46:41] Compiled: /source/sass/main.scss -> /static/css/main.min.css
Building: 92/92 (in 0:00:00)
Traceback (most recent call last):
  File "<string>", line 81, in <module>
  File "growsdk/click.core", line 664, in __call__
  File "growsdk/click.core", line 644, in main
  File "growsdk/click.core", line 991, in invoke
  File "growsdk/click.core", line 837, in invoke
  File "growsdk/click.core", line 464, in invoke
  File "growsdk/grow.commands.build", line 29, in build
  File "growsdk/grow.deployments.destinations.base", line 232, in deploy
  File "growsdk/grow.deployments.indexes", line 157, in create
  File "growsdk/encodings.utf_8", line 16, in decode
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2026' in position 11744: ordinal not in range(128)` 
```

I've tried to adjust original grow.io theme. you can find compressed folder here _link_removed._ Hope it would help.

Thanks.

**Comment by jeremydw** [comment]
Hi @nicloay -- thanks for sending this over. Indeed there's something going on with your git log that creates this issue when certain characters are present in commit messages or filenames.

I debugged it a bit and fixed the problem here in this commit –
https://github.com/grow/grow/commit/c8552f726273622954f0c0d86c1f1afd07916726

So when we do the next release of grow, this issue will be fixed. We'll likely do a release within the next few days. Thanks for reporting this!

**Comment by nicloay** [comment]
Thanks a lot for the quick reply and the prompt solution. It's a real pleasure to work with this project =).

**Comment by jeremydw** [comment]
This was fixed in `0.0.60`. Thanks!
